app/components/UI/banner/Banner.js

```diff
@@ -0,0 +1,100 @@
+export default function Banner() {
```

Comment:
you should break this into smaller components

Reply:
thank you for the suggestion! 
done